internal/schedule_client.go

```diff
@@ -282,6 +282,14 @@ type (
 		// Deprecated - This is only for update of older search attributes. This may be removed in a future version.
 		UntypedSearchAttributes map[string]*commonpb.Payload
 
+		// VersioningOverride - Sets the versioning configuration of a specific workflow execution, ignoring current
+		// server or worker default policies. This enables running canary tests without affecting existing workflows.
+		// To unset the override after the workflow is running, use Client.UpdateWorkflowExecutionOptions.
```

Comment:
Nit:

```suggestion
		// To unset the override after the workflow is running, use [Client.UpdateWorkflowExecutionOptions].
```

There are a few other godocs that could be links

Reply:
Yes, I missed quite a few, I will made them links